query/control/controller_test.go

```diff
@@ -207,6 +207,8 @@ func TestController_QueryRuntimeError(t *testing.T) {
 
 			q, err := ctrl.Query(context.Background(), &mock.Compiler{
 				CompileFn: func(ctx context.Context) (flux.Program, error) {
+					// ensure we have non-zero compile time
+					time.Sleep(1*time.Millisecond)
```

Comment:
I have seen this test fail a few times with the error below: `"expected compile duration to be above zero"`